common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -301,14 +301,11 @@ def refresh_children(self, request, suffix, update_db=True):  # pylint: disable=
         This method will update this block's 'source_libraries' field to store
         the version number of the libraries used, so we easily determine if
         this block is up to date or not.
-
-        If update_db is True (default), this will explicitly persist the changes
```

Comment:
I think that's because I pointed out to him that there was no obv use case nor test for `update_db=False`